cognite/client/_api/vision.py

```diff
@@ -0,0 +1,124 @@
+from typing import Any, Dict, List, Optional, Type, Union
+
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes.contextualization import T_ContextualizationJob
+from cognite.client.data_classes.vision import FeatureParameters, InternalId, VisionExtractJob, VisionFeature
+from cognite.client.utils._auxiliary import assert_type, to_camel_case
+from cognite.client.utils._identifier import IdentifierSequence
+
+
+class VisionAPI(APIClient):
+    _RESOURCE_PATH = "/context/vision"
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
You may remove the entire init method

Reply:
[438e9c5](https://github.com/cognitedata/cognite-sdk-python/pull/1014/commits/438e9c5c75dfcd8760cb7a4aaabd488ed55d14c4)